src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -86,11 +86,15 @@ private static MIBreakpointState StringToBreakpointState(string addr)
 
         internal static async Task<BindResult> Bind(string functionName, DebuggedProcess process, string condition, AD7PendingBreakpoint pbreak)
         {
+            process.VerifyNotDebuggingCoreDump();
```

Comment:
I meant that all upstream callers within the MIEngine assemblies catch the exception and then return the result code. Thus the exception should never reach OpenDebugAD7. Sorry for the ambiguity.